src/views/portals/utils/DashboardArticleLink.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <li>
+    <document-link :document="article" class="dashboard-link has-text-normal is-flex has-hover-background">
+      <span class="dashboard-link-main">
+        <document-title :document="article" class="document-title" />
```

Comment:
There is a problem with the lang when using this component in home page, because the lang isn't accessible in the rute params, so defaults to english.

Reply:
I don't understand what you mean, lang seems accessible.
![image](https://user-images.githubusercontent.com/24532066/228358136-ed9e78a5-2006-4e3d-bd74-eb39d5ba2cc0.png)

Reply:
It does us the route param, not hte document value.